cmd/snapctl/main.go

```diff
@@ -29,6 +29,8 @@ import (
 	"github.com/snapcore/snapd/usersession/xdgopenproxy"
 )
 
+type launcherModifiers = xdgopenproxy.LauncherModifiers
```

Comment:
I would remove this type alias, and just use the real type name below. We mainly use type aliases to make private symbols visible for tests, rather than just to save a few keystrokes.

Reply:
There's my nooby Go moment :), I think I noticed later that declaring types seems to apply to a package level and not just the scope of the file, so sure I'll change this.